t/ui/18-tests-details.t

```diff
@@ -446,9 +446,7 @@ subtest 'misc details: title, favicon, go back, go to source view, go to log vie
         'on src page for installer_timezone test'
     );
     is($driver->find_element('.cm-comment')->get_text(), '#!/usr/bin/env perl', 'we have a perl comment');
-    $driver->go_back();    # to 99946
-    $driver->find_element_by_link_text('Logs & Assets')->click;
-    wait_for_ajax msg => 'logs and assets tab';
+    $driver->get('/tests/99946/downloads_ajax');
```

Comment:
As you stated this to be a workaround I suggest to put the explanation from the git commit message as comment in here